### PR TITLE
Add 'vectorial' package

### PR DESCRIPTION
**Pull request by rbsheth**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://ci.appveyor.com/project/rbsheth/hunter/builds/21133509
  * https://travis-ci.com/rbsheth/hunter/builds/95332219

* I have submitted CI configs to https://github.com/ingenue/hunter targeting `pkg.template` branch,
  see this merged pull request https://github.com/ingenue/hunter/pull/316

* My change will resolve this "New package" request: https://github.com/ruslo/hunter/issues/1677

### CODE REVIEW

File: docs/packages/pkg/vectorial.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+.. _pkg.vectorial:
+
+vectorial
+=========
+
+-  `Official <https://github.com/scoopr/vectorial>`__
+-  `Hunterized <https://github.com/hunter-packages/vectorial>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/vectorial/CMakeLists.txt>`__
+-  Added by `Rahul Sheth <https://github.com/rbsheth>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```
[the remark this answers is not in the document]
Comment (rbsheth) [reply docs/packages/pkg/vectorial.rst]: Yeah, yeah. Done 😄

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/builds/21134741
* https://travis-ci.org/ingenue/hunter/builds/470274778?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/builds/21135305
* https://travis-ci.org/ingenue/hunter/builds/470283371?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.80

**Pull request merged**